src/spikeinterface/core/sparsity.py

```diff
@@ -383,42 +388,159 @@ def compute_sparsity(
     by_property=None,
 ):
     """
-        Get channel sparsity (subset of channels) for each template with several methods.
+    Get channel sparsity (subset of channels) for each template with several methods.
 
-        Parameters
-        ----------
-        waveform_extractor: WaveformExtractor
-            The waveform extractor
+    Parameters
+    ----------
+    templates_or_waveform_extractor: Templates | WaveformExtractor
+        A Templates or a WaveformExtractor object.
+        Some method accept both objects ("best_channels", "radius", )
+        Other method need WaveformExtractor because internally the recording is needed.
```

Comment:
```suggestion
        Some methods accept both objects (e.g. "best_channels", "radius", )
        Other methods need WaveformExtractor because internally the recording is needed.
```